src/app/modules/group/components/pending-request/pending-request.component.ts

```diff
@@ -172,33 +52,9 @@ export class PendingRequestComponent implements OnInit, OnChanges, OnDestroy {
     if (this.selection.length === 0 || this.state !== 'ready') {
       return;
     }
-
-    this.state = action === Action.Accept ? 'accepting' : 'rejecting';
-
-    this.processRequests(action, this.selection)
-      .subscribe(
-        result => {
-          this.state = 'ready';
-
-          this.displayResponseToast(
-            this.parseResults(result),
-            action === Action.Accept ? 'accept' : 'reject',
-            action === Action.Accept ? 'accepted' : 'declined'
-          );
-
-          this.selection = [];
-
-          this.dataFetching.next({
-            groupId: this.groupId,
-            includeSubgroup: this.includeSubgroup,
-            sort: this.currentSort,
-          });
-        },
-        err => {
-          this.state = 'ready';
-          this.processRequestError(err);
-        }
-      );
+    this.state = 'processing';
```

Comment:
If is it is input, you are not supposed to change it in the component (and I supposed the changed is not passed to the parent component)

Reply:
fixed